### PR TITLE
Lint: cleanup white noise

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

**What type of PR is this?**

/kind cleanup

### CODE REVIEW

File: docs/proposals/reliable-message-delivery.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ AckMessage.Operation = "response"
 We use K8s CRD to save the resourceVersion of objects that have been successfully persisted to the edge.
 
 We designed two types of CRD to save the resourceVersion. ClusterObjectSync is used to save the cluster
-scoped object and ObjectSync is used to save the namespace scoped object. 
+scoped object and ObjectSync is used to save the namesapce scoped object.
```
Comment (kevin-wangzefeng) [inline docs/proposals/reliable-message-delivery.md:152]:
```suggestion
scoped object and ObjectSync is used to save the namespace scoped object.
```

File: docs/setup/kubeedge_install_keadm.md [hunk 2]
```diff
@@ -182,10 +182,10 @@ Execute `keadm join <flags>`
     -t, --token string                        Used for edge to apply for the certificate
   ```
 
-**IMPORTANT NOTE:** 
-1. For this command `--cloudcore-ipport` flag is a mandatory flag
+**IMPORTANT NOTE:**
+1. For this command `--cloudcore-ipport` flag is a mandatory flag.
 1. If you want to apply certificate for edge node automatically, `--token` is needed.
-1. The KubeEdge version used in cloud and edge side should be same. 
+1. The kubeedge version used in cloud and edge side should be same.
```
Comment (kevin-wangzefeng) [inline docs/setup/kubeedge_install_keadm.md:188]:
```suggestion
1. The KubeEdge version used in cloud and edge side should be same.
```

File: mappers/modbus_mapper/deployment.yaml [hunk 3]
```diff
@@ -30,10 +30,9 @@ spec:
...
           mountPath: /opt/src/dpl
       nodeSelector:
-        modbus: "true"      
+        modbus: "true"
       volumes:
       - name: dpl-config-volume
         configMap:
           name:  device-profile-config-<edge_node_name>
       restartPolicy: Always
-
```
Comment (kevin-wangzefeng) [inline mappers/modbus_mapper/deployment.yaml:39]: No newline at end of file?

Comment (daixiang0) [reply mappers/modbus_mapper/deployment.yaml:39]: The vim would auto remove newline at end of file by setting, it is what text editor does, add it.

**Comment by kevin-wangzefeng** [comment]
PR has conflicts

**Comment by daixiang0** [comment]
@kevin-wangzefeng done.

File: docs/mappers/bluetooth_mapper.md [hunk 4]
```diff
@@ -159,7 +159,11 @@ perform the event once.
 
  ### Configuration File
 
+<<<<<<< HEAD
  The user can give the configurations specific to the bluetooth device using configurations provided in the configuration file present at $GOPATH/src/github.com/kubeedge/kubeedge/mappers/bluetooth_mapper/configuration/config.yaml.
+=======
+ The user can give the configurations specific to the bluetooth device using configurations provided in the configuration file present at $GOPATH/src/github.com/kubeedge/kubeedge/device/bluetooth_mapper/configuration/config.yaml.
+>>>>>>> Lint: cleanup white noise
```
Comment (kevin-wangzefeng) [inline docs/mappers/bluetooth_mapper.md:166]:
```suggestion
 The user can give the configurations specific to the bluetooth device using configurations provided in the configuration file present at $GOPATH/src/github.com/kubeedge/kubeedge/mappers/bluetooth_mapper/configuration/config.yaml.

```

**Pull request merged**